atlasdb-commons/src/main/java/com/palantir/util/MathUtils.java

```diff
@@ -244,4 +246,20 @@ public static double keepSignificantFigures(double input, int sigfigs) {
     }
 
     private MathUtils() {/**/}
+
+    public static double confidenceDistributionIsNotUniform(List<Long> values) {
```

Comment:
I read `confidenceDistributionIsNotUniform` as "confidence-distribution is not uniform" and expect it to return a boolean.  Consider renaming to `getConfidenceThatDistributionIsNotUniform `.  Likewise for the reverse method